scripts/release_gen.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import re
+import shlex
+import subprocess
+import sys
+import time
+from datetime import datetime, timezone
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional, Sequence, Tuple
+
+
+def header(msg: str) -> None:
```

Comment:
Why is this the first thing in the file? Feels like it should be near the bottom?